src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/LanguageServerWrapper.java

```diff
@@ -764,6 +804,7 @@ void registerCapability(RegistrationParams params) {
                     serverCapabilities.setDocumentRangeFormattingProvider(documentRangeFormattingProvider.getRight());
                     addRegistration(reg, () -> serverCapabilities.setDocumentRangeFormattingProvider(documentRangeFormattingProvider ));
                 }
+            //} else if ("textDocument/codeAction".equals(reg.getMethod())) { //$NON-NLS-1$
```

Comment:
What's the reason for this comment? Or mainly ported for parity?

Reply:
I just copied it over in case I needed it for the current work item.